### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by jviquerat**

### CODE REVIEW

File: pbo/src/discrete.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        cat_params = self.net.call(x)
+        cat_params = np.asarray(cat_params)[0] # take first worker only for extracting the params
+
+        # Build pdfs
+        pdfs  = [None]*self.act_dim
+        index = 0
+        for i in range(self.act_dim):
+            pdfs[i] = tfd.Categorical(cat_params[index:index + self.param_dim[i]])
+            index  += self.param_dim[i]
+
```
Comment (jviquerat) [inline pbo/src/discrete.py:69]: @aurelegoetz it seems to me that self.param_dim is not a list ?

File: pbo/src/discrete.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        self.avg_rwd = np.zeros( self.n_gen,              dtype=np.float64)
+
+    # Get actions from network
+    def get_actions(self, state, n):
+
+        # Predict
+        x  = tf.convert_to_tensor([state[0]], dtype=tf.float64)
+        cat_params = self.net.call(x)
+        cat_params = np.asarray(cat_params)[0] # take first worker only for extracting the params
+
```
Comment (jviquerat) [inline pbo/src/discrete.py:62]: @aurelegoetz could you elaborate on this comment about the "first worker" ?

File: pbo/src/discrete.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        self.n_cpu      = params.n_cpu
+
+        self.param_gen        = params.param_gen
+        self.batch            = params.batch
+        self.lr               = params.lr
+        self.epochs           = params.epochs
+        self.adv_clip         = params.adv_clip
+        self.adv_decay        = params.adv_decay
+        self.continuity_regul = params.continuity_regul
+
```
Comment (jviquerat) [inline pbo/src/discrete.py:27]: @aurelegoetz I could not find a provided value of continuity_regul in the discrete json example file, could you provide it ?

File: pbo/src/discrete.py (hunk 1 above)
Comment (aurelegoetz) [reply pbo/src/discrete.py:69]: It is, check in sphere_2d_discrete.py at line 18: 
self.act_size = [len(val) for val in self.x_values]
which is then passed to the constructor of the discrete object.

File: pbo/src/discrete.py (hunk 2 above)
Comment (aurelegoetz) [reply pbo/src/discrete.py:62]: I don't know what I meant at the time, it is a simple conversion to np.array that I am doing here. 
I must have confused that with the fact that we take state[0] to get the parameters of the probability distribution: there we take the first "worker" in the sense of the first parallel env.
I am going to change the comment accordingly.

File: pbo/src/discrete.py (hunk 3 above)
Comment (aurelegoetz) [reply pbo/src/discrete.py:27]: I'll provide it in the next commit.

File: pbo/src/discrete.py (hunk 1 above)
Comment (jviquerat) [reply pbo/src/discrete.py:69]: Indeed, I missed that

**Pull request merged**